.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_API_KEY = "api_key=0decbb49ed9b3d0c6017d7721a14c106"
```

Comment:
Please rename this file to `.env.local` or add `.env` to gitignore. The whole purpose of using .env files is to not push sensitive information from git repository.